pytket/tests/classical_test.py

```diff
@@ -1418,12 +1418,9 @@ def test_flatten_registers_with_classical_exps() -> None:
 def test_box_equality_check() -> None:
     exp1 = Bit(2) & Bit(3)
     exp2 = Bit(1) & Bit(3)
-    exp3 = Bit(1)
```

Comment:
Removed this test because it doesn't fit with the typing of ClassicalExpBox anymore (was `Any` before). As far as I can tell, the only way to add a classical exp box to a circuit is through `circ.add_classicalexpbox_bit/register` and these functions cannot support `Bit` or `BitRegister` as arguments. So I think it doesn't make sense to use a `Bit` or `Bitregister` in the constructor. 

But maybe I'm missing something?